src/main/webapp/i18n/de/tutorialGroups.json

```diff
@@ -90,7 +90,8 @@
                 "additionalInformationNotSet": "Keine zusätzlichen Informationen festgelegt"
             },
             "registeredStudents": {
-                "title": "Registrierte Studierende"
+                "title": "Registrierte Studierende",
+                "capacityWarning": "Die Kapazität der Übungsgruppe von {{ capacity }} ist erreicht! Are you sure you want to register more students?"
```

Comment:
You should stick to one language :P

Reply:
Haha :D Good catch! Removed english part in f25f24093da371c004de88e8a89d4984b33675b7